modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -4,123 +4,68 @@ include::_attributes.adoc[]
-
-=== Regular Validations
-
-Regular validations enable applications to compare a request with API specifications.  As part of the regular validations for REST or RAML proxies, the payload (schema), query parameters, URI parameters, headers, and form parameters are validated. For WSDL proxies, the envelope is validated against the structure provided in the WSDL file. If the proxy receives a parameter that is not specified in the definition, that parameter is irrespectively sent to the backend service.
+The API proxy is agnostic to where and using which programming language your backend is implemented. Additionally, your backend can be a non-Mule application.
 
+== When to Use API Proxies
 
-=== Strict Validations
+You can use API proxies:
 
-Strict validations enable applications to compare a request with proxy specifications and allow only those query parameters that are specified in the API definition. Currently, you can perform strict validations for only query parameters. This feature, which is disabled by default, can be enabled only for RAML and WSDL proxies in Mule 4. 
+* If you already have your API live, but is not yet hosted in a Mule runtime engine (Mule) server.
+* If you already have a Mule application, which is closed-code, and don’t have access to include an xref:configure-autodiscovery-4-task[Autodiscovery] for it. 
+* If you want to perform schema validation on every incoming request for a RAML, OAS, or SOAP API.
 
-The following tables describe the default settings for API proxies based on different Mule versions:
+Depending on the type of specification you used to create for your API, you can apply one of the appropriate API proxies:
 
-[[default-setting-mule-3]]
-==== Default Setting by API Proxy Versions for Mule 3.x Applications
+* HTTP or xref:building-https-proxy.adoc[HTTPS]
+* xref:building-soap-proxy.adoc[WSDL]
+* xref:proxy-deploy-raml-oas-proxy.adoc[RAML or OAS]
 
-[%header%autowidth.spread]
-|===
-| Mule Version | API Proxy | Default Setting | UI Support 
-| 3.7x | RAML or WDSL | Disabled | No
-| 3.8x | RAML or WDSL | Disabled | Yes
-| 3.9x | RAML or WDSL | Disabled | Yes
-|===
 
-Mule 3.x applications and API proxy compatibility synopsis:
+== How API Proxies Work
 
-* Only regular proxy validations are available for Mule 3.x applications.
-* Validations are performed for only WSDL and RAML proxies.
-* Proxies are not versioned in Mule 3.x applications.
-* You can edit proxies from API Manager for Mule versions 3.8.x and later.
+You can download a preconfigured Mule application, if you are running an on-premise Mule instance, and deploy it just as any other Mule application by simply xref:org-credentials-config-mule4.adoc[configuring your Mule instance with the correct Anypoint Platform credential].
```

Comment:
GLOBAL: Always use "premises," plural, whether using the adjective form (on-premises) or the adverb form "on premises")

Reply:
```suggestion
You can download a preconfigured Mule application, if you are running an on-premise Mule instances, and deploy it in the same way as any other Mule application, by xref:org-credentials-config-mule4.adoc[configuring your Mule instance with the correct Anypoint Platform credential].
```

Reply:
Done! Thanks!